src/main/java/org/team1540/robot2018/commands/auto/sequences/SwitchDoubleCube.java

```diff
@@ -0,0 +1,71 @@
+package org.team1540.robot2018.commands.auto.sequences;
+
+import static org.team1540.robot2018.commands.wrist.CalibrateWrist.CalibratePosition.OUT;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+import edu.wpi.first.wpilibj.command.TimedCommand;
+import org.team1540.base.util.SimpleCommand;
+import org.team1540.robot2018.Robot;
+import org.team1540.robot2018.Tuning;
+import org.team1540.robot2018.commands.arms.DropCube;
+import org.team1540.robot2018.commands.elevator.MoveElevator;
+import org.team1540.robot2018.commands.intake.AutoIntake;
+import org.team1540.robot2018.commands.intake.Eject;
+import org.team1540.robot2018.commands.wrist.CalibrateWristMP;
+import org.team1540.robot2018.motion.FollowProfile;
+
+public class SwitchDoubleCube extends CommandGroup {
+  public SwitchDoubleCube(String side) {
+
+    // Lower the wrist
+    addSequential(new CalibrateWristMP(OUT));
+
+    addSequential(new CommandGroup() {
+      {
+        // Go to the switch and raise elevator at the same time
+        addParallel(new FollowProfile("middle_to_" + side + "_switch_multi"));
+        addSequential(new MoveElevator(false, Tuning.elevatorFrontSwitchPosition));
+      }
+    });
+    // After arrival, eject the cube
+    addSequential(new Eject(Tuning.intakeEjectSpeedAuto, 0.5));
+
+    addSequential(new CommandGroup() {
+      {
+        // Go back to the middle and move elevator down and recalibrate the wrist
+        addSequential(new FollowProfile(side + "_switch_to_middle_multi"));
```

Comment:
Should not be sequential. However, removing this might cause wrist to hit the switch fence–should be tested